src/client/components/AuthBar.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const AuthBar = (props) => {
+  const user = props.user.user;
+  const name = user ? user.name : '';
+
+  return (
+    <span>{name || <a href="/login">Login</a>}</span>
+  );
+};
+
+AuthBar.propTypes = {
+  user: React.PropTypes.object,
```

Comment:
what happens when `props.user` doesn't exist? so should this add `.isRequired`? or test at line 4 for `props.user && props.user.user`.